pyxem/utils/expt_utils.py

```diff
@@ -419,6 +420,103 @@ def reference_circle(coords, dimX, dimY, radius):
     return img
 
 
+def _find_peak_max(arr: np.ndarray, sigma: int, m: int=50, w: int=10, kind: int=3) -> float:
+    """Find the index of the pixel corresponding to peak maximum in 1D pattern
+
+    Parameters
+    ----------
+    sigma : int
+        Sigma value for Gaussian blurring kernel for initial beam center estimation.
+    m : int
```

Comment:
Can we call this something more self documenting please? e.g. interpolation_factor